css/slider.css

```diff
@@ -0,0 +1,204 @@
+.slider-main{
+    display: flex;
+    margin: 0 auto;
+    width: 100%;
+    overflow: hidden;
+    height: 500px;
+    position: relative;
+}
+
+.slider-item-buttons{
+    display: flex;
+    align-items: center;
+    height: 100%;
+    position: absolute;
+    z-index: 1;
+    opacity: 0;
+    transition: .3s;
+}
+
+.slider-item-buttons button{
+    border: none;
+    background-color: #d3d3d3;
+    transition: .3s;
+    border-radius: 3px;
+    font-size: 1.7rem;
+    opacity: .5;
+}
+
+.slider-item-buttons.left{
+    left: 0;
+}
+.slider-item-buttons.right{
+    right: 0;
+}
+
+.slider-main:hover .slider-item-buttons{
+    opacity: 1;
+    transition: .3s;
+}
+
+.slider-item-buttons button:hover{
+    color: #ffffff;
+    background-color: #ff8c00;
+    transition: .3s;
+    opacity: 1;
+}
+
+.slider-item-container{
+    margin: 0 auto;
+    width: 100%;
+    overflow: hidden;
+    height: 500px;
+    position: relative;
+}
+
+.slider-item-background{
+    background-size: cover;
+    background-position: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    position: absolute;
+    transition: .6s;
+}
+
+.slider-item-background.left{
+    left: -100%;
+    transition: .6s;
+}
+
+.slider-item-background.right{
+    left: 100%;
+    transition: .6s;
+}
+
+.slider-item-background.current{
+    left: 0;
+    transition: .6s;
+}
+
+.slider-item{
+    position: relative;
+    color: #ffffff;
+    text-shadow: 0px 0px 1px black;
```

Comment:
when value is 0 - where couldn't be any measurements. 0% and 0px it's a zero.
so don't write measure.
use just "0"